codes/go/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/reanon/hello-algo/codes/go
```

Comment:
Is this file necessary?
It looks like pointing out the local environment.

Reply:
Go application project recommend module named as `github.com/YOUR-USER-OR-ORG-NAME/YOUR-REPO-NAME`. It will be replaced with  github.com/krahets/hello-algo/codes/go in the next patch

Reply:
I see. By the way, I will add a file header for each code file to reveal the file name, created time, and author.

For instance,

```java
/*
 * File: leetcode_two_sum.java
 * Created Time: 2022-11-25
 * Author: Krahets (krahets@163.com)
 */
```

You shall replace the content with yours in the consequent commits, thanks!

Reply:
I get it, file header will be append like this in golang: 
```
// File: leetcode_two_sum.go
// Created Time: 2022-11-25
// Author: Krahets (krahets@163.com)
```

Reply:
Thanks @Reanon